elasticsearch/Dockerfile

```
@@ -10,8 +10,8 @@ COPY bin/ bin/
 # https://docs.search-guard.com/latest/search-guard-versions
 # https://search.maven.org/search?q=g:com.floragunn%20AND%20a:search-guard-6&core=gav
 ARG ELK_VERSION
-ENV SG_VERSION=${ELK_VERSION}-23.0
-RUN elasticsearch-plugin install --batch com.floragunn:search-guard-6:${SG_VERSION} \
+ARG SG_VERSION
```

```suggestion
ARG SG_VERSION
ENV SG_VERSION=${SG_VERSION}
```
As ugly as it may seem this extra `ENV` is [necessary|https://docs.docker.com/engine/reference/builder/#using-arg-variables] in order to be able to use a build argument inside a `RUN` instruction.